queries/liquidsoap/locals.scm

```diff
@@ -0,0 +1,15 @@
+[(anonymous_function) (binding) (def) (let)] @scope
+
+(anonymous_argument (var) @local.parameter)
+(labeled_argument label: (var) @local.parameter)
+
+(binding
+  defined: (var) @local.var)
+(def defined: (var) @local.var)
+(let defined: (var) @local.var)
+(meth_pattern (var) @local.var)
+(list_pattern (var) @local.var)
+(tuple_pattern (var) @local.var)
+(spread (var) @local.var)
```

Comment:
you never changed these to what they should be (in CONTRIBUTING.md) so CI is failing

Reply:
Ha darn sorry about that I think I did through GH and it got force-pushed out.